lib/team_statistics.rb

```diff
@@ -0,0 +1,6 @@
+require_relative "./stat_tracker"
```

Comment:
@Jonathan-M-Wilson I didn't see a branch for yours, but here is the gist of how the file should look before you add your code